src/Orleans/Runtime/OutsideRuntimeClient.cs

```diff
@@ -90,47 +90,37 @@ public IStreamProviderRuntime CurrentStreamProviderRuntime
 
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Reliability", "CA2000:Dispose objects before losing scope",
             Justification = "MessageCenter is IDisposable but cannot call Dispose yet as it lives past the end of this method call.")]
-        public OutsideRuntimeClient(ClientConfiguration cfg, bool secondary = false)
+        public OutsideRuntimeClient()
         {
-            if (cfg == null)
+            this.handshakeClientId = GrainId.NewClientId();
+            tryResendMessage = TryResendMessage;
+            unregisterCallback = msg => UnRegisterCallback(msg.Id);
+            callbacks = new ConcurrentDictionary<CorrelationId, CallbackData>();
+            localObjects = new ConcurrentDictionary<GuidId, LocalObjectData>();
+        }
+
+        internal void ConsumeServices(IServiceProvider services)
```

Comment:
Maybe, but there are a lot of circular dependencies back to IRuntimeClient, so this was the easiest approach. I'll see what it takes to sever those dependencies.